dependencies.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-    minSdkVersion = 14
+    minSdkVersion = 16
```

Comment:
Why did this change?

Reply:
`minSdkVersion` has to 15 for 310abp so i figured might as well update it to the latest supported